packages/core/src/components/text-field/readme.md

```diff
@@ -14,15 +14,17 @@
 | `helper`        | `helper`         | Helper text                                 | `string`                              | `undefined`  |
 | `label`         | `label`          | Label text                                  | `string`                              | `''`         |
 | `labelPosition` | `label-position` | Position of the label for the Text Field.   | `"inside" \| "no-label" \| "outside"` | `'no-label'` |
+| `max`           | `max`            | Max allowed value for input type number     | `number \| string`                    | `undefined`  |
 | `maxLength`     | `max-length`     | Max length of input                         | `number`                              | `undefined`  |
+| `min`           | `min`            | Min allowed value for input type number     | `number \| string`                    | `undefined`  |
 | `modeVariant`   | `mode-variant`   | Mode variant of the Text Field              | `"primary" \| "secondary"`            | `null`       |
 | `name`          | `name`           | Name property                               | `string`                              | `''`         |
 | `noMinWidth`    | `no-min-width`   | Unset minimum width of 208px.               | `boolean`                             | `false`      |
 | `placeholder`   | `placeholder`    | Placeholder text                            | `string`                              | `''`         |
 | `readOnly`      | `read-only`      | Set input in readonly state                 | `boolean`                             | `false`      |
 | `size`          | `size`           | Size of the input                           | `"lg" \| "md" \| "sm"`                | `'lg'`       |
 | `state`         | `state`          | Error state of input                        | `"default" \| "error" \| "success"`   | `'default'`  |
-| `type`          | `type`           | Which input type, text, password or similar | `"password" \| "text"`                | `'text'`     |
+| `type`          | `type`           | Which input type, text, password or similar | `"number" \| "password" \| "text"`    | `'text'`     |
```

Comment:
Left comment in PR but to me it's bit confusing to have text-field type="number".

Reply:
If we'd separate it into another component we wouldn't need to have put the caveat that `min`/`min` only is valid for type `number`

Reply:
See: https://github.com/scania-digital-design-system/tegel/pull/498#issuecomment-1920893280